client/webserver/site/src/js/markets.js

```diff
@@ -400,11 +403,30 @@ export default class MarketsPage extends BasePage {
   handleBookRoute (data) {
     const market = this.market
     const page = this.page
-    const [b, q] = [market.base, market.quote]
-    if (data.base !== b.id || data.quote !== q.id) return
```

Comment:
This check is necessary since we are using WebSockets for order book feeds and cannot count on ordered responses.